redis-plugin/src/main/java/org/entando/entando/plugins/jpredis/aps/system/redis/condition/RedisSentinelCondition.java

```diff
@@ -0,0 +1,30 @@
+package org.entando.entando.plugins.jpredis.aps.system.redis.condition;
+
+import org.entando.entando.plugins.jpredis.aps.system.redis.RedisEnvironmentVariables;
+import org.springframework.context.annotation.Condition;
+import org.springframework.context.annotation.ConditionContext;
+import org.springframework.core.type.AnnotatedTypeMetadata;
+import org.springframework.util.MultiValueMap;
+
+public class RedisSentinelCondition implements Condition {
```

Comment:
Maybe a name like `ConditionRedisSentinelActive` it would be more self-explained ...